builtin/providers/aws/resource_aws_elb.go

```diff
@@ -213,38 +231,16 @@ func resourceAwsElbCreate(d *schema.ResourceData, meta interface{}) error {
 
 	d.Set("tags", tagsToMapELB(tags))
 
-	if d.HasChange("health_check") {
-		vs := d.Get("health_check").(*schema.Set).List()
-		if len(vs) > 0 {
-			check := vs[0].(map[string]interface{})
-
-			configureHealthCheckOpts := elb.ConfigureHealthCheckInput{
-				LoadBalancerName: aws.String(d.Id()),
-				HealthCheck: &elb.HealthCheck{
-					HealthyThreshold:   aws.Integer(check["healthy_threshold"].(int)),
-					UnhealthyThreshold: aws.Integer(check["unhealthy_threshold"].(int)),
-					Interval:           aws.Integer(check["interval"].(int)),
-					Target:             aws.String(check["target"].(string)),
-					Timeout:            aws.Integer(check["timeout"].(int)),
-				},
-			}
-
-			_, err = elbconn.ConfigureHealthCheck(&configureHealthCheckOpts)
-			if err != nil {
-				return fmt.Errorf("Failure configuring health check: %s", err)
-			}
-		}
-	}
-
 	return resourceAwsElbUpdate(d, meta)
 }
 
 func resourceAwsElbRead(d *schema.ResourceData, meta interface{}) error {
 	elbconn := meta.(*AWSClient).elbconn
+	ELBName := d.Id()
```

Comment:
Prefer lower case names, so `elbName` here. The upper case typically implies a thing that's exported